src/Components/TodoContainer.js

```diff
@@ -1,74 +1,124 @@
+            method: "GET",
+            headers: {
+                Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+            },
+        };
+
+        // Construct the URL for the API with ascending or descending order
+        // Update the URL with sort parameters (field and direction)
+        const order = isAscending ? "asc" : "desc";
+        const sortField = "created"; // Change this to the desired sorting field
+        const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}?sort[0][field]=${sortField}&sort[0][direction]=${order}`;
+
+        try {
+            const response = await fetch(url, options);
+
+            if (!response.ok) {
+                console.log("Airtable error response:");
+                console.log(response);
+                throw new Error(`Error: ${response.status}`);
+            }
+
+            // Parse the response JSON
+            const data = await response.json();
+            console.log('API Response Data:', data);
+
+            // Transform API data.records into todo objects
+            const todos = data.records.map((record) => ({
+                id: record.id,
+                title: record.fields.title,
+                createdTime: new Date(record.createdTime),
+            }));
+
+            // Update todoList and isLoading states
+            setTodoList(todos);
+            setIsLoading(false);
+        } catch (error) {
+            console.log(`Fetch error: ${error.message}`);
+        }
+    }, [isAscending]);
+
+    // Fetch data on initial render and whenever sorting order changes
+    useEffect(() => {
+        fetchData();
+        // eslint-disable-next-line 
+    }, []);
+
+    // Function to toggle sorting order and sort the todoList accordingly
+    function toggleSortingOrder() {
+        // Toggle between ascending and descending order
+        setIsAscending(!isAscending);
+
+        // Sort the todoList based on the new sorting order
+        setTodoList((prevTodoList) =>
+            [...prevTodoList].sort((a, b) => {
+                if (isAscending) {
+                    return a.createdTime < b.createdTime ? -1 : 1;
+                } else {
+                    return a.createdTime > b.createdTime ? -1 : 1;
+                }
+            })
+        )
+    }
+
+
+    function removeTodo(id) {
+        // Construct the URL for the DELETE request
+        const deleteUrl = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/${id}`;
+
+        // Define options for the DELETE request
+        const options = {
+            method: "DELETE",
+            headers: {
+                Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+            },
+        };
+
+        // Send the DELETE request
+        fetch(deleteUrl, options)
+            .then((response) => {
+                if (response.status === 204) {
```

Comment:
awesome, thank you for adding this! 

there's a small typo here that was causing the frontend to throw an error even when the backend had succeeded, because we're checking `204` instead of `200` which is the "everything worked" status code. 
```suggestion
                if (response.status === 200) {
```